crates/sui-bridge/src/tools/mod.rs

```diff
@@ -245,33 +249,111 @@ pub struct BridgeCliConfig {
     /// Proxy address for BridgeCommittee deployed on Eth
     pub eth_bridge_committee_proxy_address: EthAddress,
     /// Proxy address for BridgeLimiter deployed on Eth
     pub eth_bridge_limiter_proxy_address: EthAddress,
-    /// Path of the file where bridge client key (any SuiKeyPair) is stored as Base64 encoded `flag || privkey`.
-    /// The derived accounts
-    pub bridge_client_key_path_base64_sui_key: PathBuf,
+    /// Key pair for Sui operations
+    sui_key: SuiKeyPair,
+    /// Key pair for Eth operations, must be Secp256k1 key
+    // pub eth_key: SuiKeyPair,
+    eth_signer: EthSigner,
 }
 
-impl Config for BridgeCliConfig {}
+impl LoadedBridgeCliConfig {
+    pub async fn load(cli_config: BridgeCliConfig) -> anyhow::Result<Self> {
+        if cli_config.eth_key_path.is_none() && cli_config.sui_key_path.is_none() {
+            return Err(anyhow!(
+                "At least one of `sui_key_path` or `eth_key_path` must be provided"
+            ));
+        }
+        let sui_key = if let Some(sui_key_path) = &cli_config.sui_key_path {
+            Some(read_key(sui_key_path, false)?)
+        } else {
+            None
+        };
+        let eth_key = if let Some(eth_key_path) = &cli_config.eth_key_path {
+            let eth_key = read_key(eth_key_path, true)?;
+            Some(eth_key)
+        } else {
+            None
+        };
+        let (eth_key, sui_key) = {
+            if eth_key.is_none() {
+                let sui_key = sui_key.unwrap();
+                if !matches!(sui_key, SuiKeyPair::Secp256k1(_)) {
+                    return Err(anyhow!("Eth key must be an ECDSA key"));
+                }
+                (sui_key.copy(), sui_key)
+            } else if sui_key.is_none() {
+                let eth_key = eth_key.unwrap();
+                (eth_key.copy(), eth_key)
+            } else {
+                (eth_key.unwrap(), sui_key.unwrap())
+            }
+        };
+
+        let provider = Arc::new(
+            ethers::prelude::Provider::<ethers::providers::Http>::try_from(&cli_config.eth_rpc_url)
+                .unwrap()
+                .interval(std::time::Duration::from_millis(2000)),
+        );
+        let private_key = Hex::encode(eth_key.to_bytes_no_flag());
+        let eth_signer = get_eth_signer_client(&cli_config.eth_rpc_url, &private_key).await?;
+        let sui_bridge = EthSuiBridge::new(cli_config.eth_bridge_proxy_address, provider.clone());
+        let eth_bridge_committee_proxy_address: EthAddress = sui_bridge.committee().call().await?;
+        let eth_bridge_limiter_proxy_address: EthAddress = sui_bridge.limiter().call().await?;
+
+        let eth_address = eth_signer.address();
+        let eth_chain_id = provider.get_chainid().await?;
+        let sui_address = SuiAddress::from(&sui_key.public());
+        println!("Using Sui address: {:?}", sui_address);
+        println!("Using Eth address: {:?}", eth_address);
+        println!("Using Eth chain: {:?}", eth_chain_id);
```

Comment:
is `println` the way we print something in the tool? or is this something that should be removed or done differently?